geomstats/geometry/lie_group.py

```diff
@@ -207,17 +220,17 @@ def exp_not_from_identity(self, tangent_vec, base_point, point_type=None):
         """
         if point_type == 'vector':
             jacobian = self.jacobian_translation(
-                point=base_point, left_or_right='left', point_type=point_type)
+                point=base_point, left_or_right='left')
```

Comment:
not sure to understand, do you mean there would be a problem if the user passes a different point_type from default_point_type?

Reply:
Yes. I meant, would this work for example:

```
so = SpecialOrthogonal(n=3, default_point_type='matrix')
# [...] defining vec and pt in vector representation
so.exp_not_from_identity(vec, pt, point_type='vector')
```
because it will go get the `jacobian_translation` in the `matrix` representation, whereas `vec` and `pt` are in vector representation.

I guess, my confusion is maybe on what we mean with `point_type` for the method `jacobian_translation`. I understood it as: `point_type` of the vectors on which the Jacobian acts. Therefore, it `point_type`=='vector' then jacobian_translation returns a matrix, if `point_type`=='matrix', then jacobian_translation returns... also a matrix?

Reply:
Ok I see the problem so we want to remove the point_type arg here as well? I think the best would also to implement direcly the push forward of a tangent vector by the left (or right) translation rather than just the jacobian (which always returns a matrix) because it will allow to deal with all combinations of left, right, vector, matrix in one method instead of everytime we want to go to identity!

Reply:
Yes, removing the point_type in this function sounds great: did we also decide to move to one `point_type` per object or one `point_type` per class? (maybe for another PR) removing the point_type of this function would definitely go in this direction.

Yes, agreed for implementing the push-forward directly (provided that the `jacobian_translation` matrix itself is not needed somewhere else in the code? even for another Lie group for example?)

Reply:
it can only be usefull for a Lie group represented by vectors, because if we represent by matrices, jacobian_translation(point) just returns point (and the left_right info is actually missing)